hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/RawTraceViewGeneratorTest.java

```diff
@@ -0,0 +1,40 @@
+package org.hypertrace.viewgenerator.generators;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+import org.apache.avro.file.DataFileReader;
+import org.apache.avro.specific.SpecificDatumReader;
+import org.hypertrace.core.datamodel.StructuredTrace;
+import org.hypertrace.viewgenerator.api.RawTraceView;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RawTraceViewGeneratorTest {
+  private RawTraceViewGenerator rawTraceViewGenerator;
+
+  @BeforeEach
+  public void setup() {
+    rawTraceViewGenerator = new RawTraceViewGenerator();
+  }
+
+  @Test
+  public void testGenerateView_HotrodTrace() throws IOException {
+    URL resource =
```

Comment:
I think from here to line 33 is the preparation for the test and we could abstract it into a private static function, otherwise it makes it makes noise when reading the test.